images/ose-aws-efs-csi-driver-operator.yml

```diff
@@ -12,29 +12,46 @@ content:
     ci_alignment:
       streams_prs:
         ci_build_root:
-          stream: rhel-8-golang-ci-build-root
+          stream: rhel-9-golang-ci-build-root
         auto_label:
         - approved
         - lgtm
     pkg_managers: [] # disabling cachito processing awaiting STONEBLD-1973
 distgit:
-  branch: rhaos-{MAJOR}.{MINOR}-rhel-8
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
   component: ose-aws-efs-csi-driver-operator-container
   bundle_component: ose-aws-efs-csi-driver-operator-bundle-container
 enabled_repos:
-- rhel-8-baseos-rpms
-- rhel-8-appstream-rpms
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
 for_payload: false
 from:
   builder:
   - stream: golang
-  member: openshift-enterprise-base
-name: openshift/ose-aws-efs-csi-driver-rhel8-operator
+  member: openshift-enterprise-base-rhel9
+name: openshift/ose-aws-efs-csi-driver-rhel9-operator
 name_in_bundle: aws-efs-csi-driver-rhel8-operator
```

Comment:
this looks funny, but changing it will break operator bundles. that change needs to be coordinated with upstream. they aren't likely to notice this name in their image-references file, so as far as i'm concerned we can just leave it this way until they want the change.